packages/engine/stdlib/src/py/hstd/spatial.py

```diff
@@ -13,32 +13,32 @@
 class Topology:
     x_bounds: List[float]
     y_bounds: List[float]
-    z_bounds: Optional[List[float]]
+    z_bounds: Optional[List[float]] = [0, 0]
```

Comment:
Both the `init.py` and `spatial.py` have a copy of the `Topology` class. I think it would be better to move it into a shared module (either an existing one or a new file).

Reply:
Setting a list as a default value in a dataclass is not possible because lists are mutable. Instead, import `field` from `dataclasses` and do ` = field(default_factory=lambda: [0, 0])`